_component_design/tooltip.md

```diff
@@ -13,28 +13,304 @@ last-modified: 2017-08-17
+- Tooltip should be used on icon images, text, and table rows.
+- Tooltips should be placed near the object that is being hovered over. 
+- Tooltips should never be placed in a way that interferes with what the user is doing, or obscure the object of interest.
+- The maximum width for the tooltip is 400px.
+- Do not put interactive controls inside the tooltip.
+- Don't put images that look like they are interactive inside the tooltip.
+
+
+### Problems being solved
+{: .hxSubSectionTitle}
+Tips are a powerful way to simplify a user interface. They provide information users need when they need it, with minimal effort on their part. Tips can help you use screen space more effectively and reduce screen clutter.
+
+{% endcolumn %}
+
+</div>
+
+## Default composition 
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Tooltips are composed of two primary portions, the tooltip container and the trigger:
+
+The trigger is the element that displays the tooltip component when interacted with from a user. (see trigger types section)
+
+The container houses the border, pointer, and text/content. 
+
+The the tooltip will stand off from the back ground by: a lighter background color than the interface background, a slight rule (line) around the tooltip box, and a slight shadow around the tooltip so it seems “higher” or “raised up” from the items below it. 
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Tooltip composition."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-composition.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Tooltip specs 
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Spacing
+{: .hxSubSectionTitle}
+The tooltip component has varied widths based on the amount of content contained within its bounds but always relies on the following consistent spacing specifications:
+
+- 8px between vertical elements
+- 12px between not related elements 
+- max-width of 400px
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Spacing specifications for tooltip component."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-spacing-1.svg){:width="100%"}
 
--   Activates when hovered on click.
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
--   The tooltip disappears when mouse hovers away, or when clicked in the icon.
-    Tooltip does not reactivate on click.
+### Text
+{: .hxSubSectionTitle}
 
--   Tooltip should only be used on icon images, text, and table rows; use when
-    see fit.
+- 14px Roboto Medium for headers
+- 14px Roboto Regular for body copy
```

Comment:
what?

Reply:
![image](https://user-images.githubusercontent.com/586682/31908137-888f98b2-b7fb-11e7-98d3-61448320b68b.png)

This is the output. Does that make sense?